shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -266,6 +266,11 @@ public void onLongPress(int nodeId) {
         /** The user has opened a tooltip. */
         @Override
         public void onTooltip(@NonNull String message) {
+          // Tooltip open is no longer announced after API 28 and is handled by
```

Comment:
is there a link about this? e.g. release notes from Android or something similar?

Reply:
I don't think there is release note, this is just my observation after playing around the native app

Reply:
in this case could you indicate in the comment about how to reproduce the assumption?